minizip fails to build in linux (WSL)

```
Package: minizip:x64-linux
Vcpkg version: 0.0.110-unknownhash
```

```
CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:247 (_find_package):
  By not providing "FindBzip2.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Bzip2", but
  CMake did not find one.

  Could not find a package configuration file provided by "Bzip2" with any of
  the following names:

    Bzip2Config.cmake
    bzip2-config.cmake

  Add the installation prefix of "Bzip2" to CMAKE_PREFIX_PATH or set
  "Bzip2_DIR" to a directory containing one of the above files.  If "Bzip2"
  provides a separate development package or SDK, be sure it has been
  installed.
Call Stack (most recent call first):
  CMakeLists.txt:10 (find_package)
```

--- a/ports/minizip/CMakeLists.txt
+++ b/ports/minizip/CMakeLists.txt
@@ -7,7 +7,7 @@ endif()
 
 find_package(ZLIB REQUIRED)
 
-find_package(Bzip2 REQUIRED)
+find_package(BZip2 REQUIRED)
 
 set(MIN_SRC contrib/minizip)
 

